src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Options for MSAL token caches. 
+    /// </summary>
+    /// <remarks>
+    /// These option do not allow configuration of external cache serialization, for which you should use <see cref="TokenCache.SetAfterAccessAsync(System.Func{TokenCacheNotificationArgs, System.Threading.Tasks.Task})"/> and other callbacks.
```

Comment:
I don't think the do it yourself path should be mentioned. We have Microsoft.Identity.Web.TokenCache for a reason. A link to the documentation should be enough.